frontend/src/metabase/components/DebouncedFrame.jsx

```diff
@@ -49,11 +55,11 @@ export default class DebouncedFrame extends React.Component {
       this.state.height !== nextProps.height
     ) {
       if (this.state.width == null || this.state.height == null) {
-        this.setSizeDebounced(nextProps.width, nextProps.height);
+        this.updateSizeDebounced();
       } else {
-        this.setSizeDebounced(nextProps.width, nextProps.height);
         this._transition = true;
         this._updateTransitionStyle();
+        this.updateSizeDebounced();
```

Comment:
Rearrange the order so that it reads.
1. We update the state as `transitioning`
2. We scheduled `updateSize` for later

This actually works the same as before as `updateSizeDebounced` is a debounced function, so even if it's called before `_updateTransitionStyle`, the non-debounced version would still be called after `_updateTransitionStyle` anyway. This reordering reflects that fact.